lib/activity/comments.js

```diff
@@ -2,10 +2,24 @@ const cache = require('../cache');
 const { Comment } = require('../messages/comment');
 
 module.exports = async (context, subscription, slack) => {
-  const { comment, repository } = context.payload;
+  const { repository } = context.payload;
 
   const issue = context.payload.issue || context.payload.pull_request;
 
+  let { comment } = context.payload;
+
+  // Fetch updated comment to get body_html
+  try {
+    comment = (await context.github.request({
+      method: 'GET',
+      url: context.payload.comment.url,
+      headers: { accept: 'application/vnd.github.html+json' },
+    })).data;
+  } catch (err) {
+    // Review comments API is not yet accessible by GitHub Apps
+    context.log.warn({ err }, 'Could not fetch HTML for comment');
```

Comment:
This is causing a _ton_ of exceptions in production. I'm re-deployed master and will see if we can get GitHub App support for `/repos/:owner/:repo/pulls/comments/:id`